libs/platform/kernel/ebpf_platform_kernel.c

```diff
@@ -437,7 +437,7 @@ typedef struct _ebpf_non_preemptible_work_item
 
 static void
 _ebpf_deferred_routine(
-    KDPC* deferred_procedure_call, PVOID deferred_context, PVOID system_argument_1, PVOID system_argument_2)
+    KDPC* deferred_procedure_call, void* deferred_context, void* system_argument_1, void* system_argument_2)
```

Comment:
Missing SAL annotations?  Might be worthwhile to see if we have an ebpf equivalent for the ```KDPC``` type.